CRM/Case/XMLProcessor/Report.php

```diff
@@ -477,9 +477,7 @@ public function &getActivity($clientID, $activityDAO, &$activityTypeInfo) {
     if ($activityDAO->medium_id) {
       $activity['fields'][] = array(
         'label' => ts('Medium'),
-        'value' => CRM_Core_OptionGroup::getLabel('encounter_medium',
-          $activityDAO->medium_id, FALSE
-        ),
+        'value' => CRM_Core_PseudoConstant::getLabel('CRM_Activity_BAO_Activity', 'encounter_medium', $activityDAO->medium_id),
```

Comment:
I believe this is incorrect and should be 'medium_id' instead of 'encounter_medium'

Reply:
Yes